src/js/directives/elementPosition.js

```diff
@@ -0,0 +1,66 @@
+'use strict';
+
+var angular = require('angular');
+
+module.exports = function(app) {
+
+  app.directive('elementPosition', [
```

Comment:
A little documentation would be nice. What is the directive, what does it do, how do I use it?